devex/vscode-extension/README.md

```diff
@@ -0,0 +1,383 @@
+  ```bash
+  sudo apt update
+  sudo apt -y install dkms
+  wget https://download.01.org/intel-sgx/sgx-dcap/1.4/linux/distro/ubuntuServer16.04/sgx_linux_x64_driver_1.21.bin -O sgx_linux_x64_driver.bin
+  chmod +x sgx_linux_x64_driver.bin
+  sudo ./sgx_linux_x64_driver.bin
+  ```
+
+  On Ubuntu 18.04, run:
+  ```bash
+  sudo apt update
+  sudo apt -y install dkms
+  wget https://download.01.org/intel-sgx/sgx-dcap/1.4/linux/distro/ubuntuServer18.04/sgx_linux_x64_driver_1.21.bin -O sgx_linux_x64_driver.bin
+  chmod +x sgx_linux_x64_driver.bin
+  sudo ./sgx_linux_x64_driver.bin
+  ```
+
+  Note: A new version of the driver may have been released since this
+        documentation was written. Check on
+        [Intel's Open Source site](https://01.org/intel-software-guard-extensions/downloads)
+        if a more recent SGX DCAP driver exists.
+
+* Ensure that the requirements are met for the
+  [Azure IoT Edge extension](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-edge).
+
+* For Azure IoT Edge projects, enable cross-building:
+
+  * On Ubuntu 19.04 and 18.10, this should already be enabled by having
+    installed the prerequisites listed previously.
+
+  * On Ubuntu 18.04 and 16.04 by:
+    ```bash
+    sudo mkdir -p /lib/binfmt.d
+    sudo sh -c 'echo :qemu-arm:M::\\x7fELF\\x01\\x01\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x02\\x00\\x28\\x00:\\xff\\xff\\xff\\xff\\xff\\xff\\xff\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xfe\\xff\\xff\\xff:/usr/bin/qemu-arm-static:F > /lib/binfmt.d/qemu-arm-static.conf'
+    sudo sh -c 'echo :qemu-aarch64:M::\\x7fELF\\x02\\x01\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x02\\x00\\xb7\\x00:\\xff\\xff\\xff\\xff\\xff\\xff\\xff\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xfe\\xff\\xff\\xff:/usr/bin/qemu-aarch64-static:F > /lib/binfmt.d/qemu-aarch64-static.conf'
+    sudo systemctl restart systemd-binfmt.service
+    ```
+
+  * To validate that your system is configured for cross-building, you can
+    try testing the docker containers that will be needed in the build:
+    ```bash
+    docker run arm32v7/ubuntu:xenial
+    docker run aarch64/ubuntu:xenial
+    docker run amd64/ubuntu:xenial
+    ```
+
+  * Install the [iotedgehubdev](https://pypi.org/project/iotedgehubdev/) tool:
+    ```bash
+    pip install --upgrade iotedgehubdev
+    ```
+
+* If you are seeing permission issues related to connecting to the Docker
+  daemon, add your Linux user to the docker group. This will allow your user
+  to connect and issue commands to the Docker daemon:
+  ```bash
+  sudo usermod -a -G docker $USER
+  ```
+
+### Windows
+
+* [Install Docker](https://docs.docker.com/install/)
+
+* Ensure sure that Git long paths are enabled:
+  `git config --global core.longpaths true`
+
+* Ensure Docker is configured to use Linux containers.  See Docker help
+  regarding **Switch between Windows and Linux containers**
+  [here](https://docs.docker.com/docker-for-windows/).
```

Comment:
recommend replacing "[here]" with "the [Get started with Docker for Windows] page", for accessibility reasons.  (Short version: one example is when you tab through links and a screen reader for the blind will read the text.  Tabbing into this link will just say "here" which is useless.   Other examples exist and lots more discussion is available by searching on accessibility and "click here").

Reply:
Fixed.